README.md

```diff
@@ -2,20 +2,104 @@
 
 > Test client for Xo-Server
 
-## Usage
+You should have a `admin@admin.net` account with password `admin`.
 
-xo-server needs to be installed on `localhost:9000`.
-You should also have a `admin@admin.net` account with password `admin`.
+## Adding a test
 
-There should be a Xen Server available at `192.168.1.3`with the credentials `root` and password `qwerty`.
+### Organization
 
-Warning ! This client will create and delete a lot of items (users, groups, servers ...) to test features
+```
+src
+├─ user
+|   ├─ __snapshots__
+|   |     └─ index.spec.js.snap 
+|   └─ index.spec.js
+├─ job
+¦   └─ index.spec.js
+¦
+¦
+├─ _xoWithTestHelpers.js
+└─ util.js
+```
 
+The tests can describe xo methods or scenarios:  
 ```
-> npm ci
-> npm test
+describe("user", () => {
+  // testing a method
+  describe(".create(): ", () => {
+    it("creates a user with permission", () => {
+      /* some tests */
+    });
+  });
+      
+  // testing a scenario
+  if("creates two users, modifies a user email to be the same with the other and fails trying to connect them", () => {
+    /* some tests */
+  });
+});
 ```
 
+### Test environment
```

Comment:
IMHO this section is not really relevant, simply move its content into *Best practices*.